flixel/text/FlxBitmapText.hx

```diff
@@ -1461,7 +1462,7 @@ class FlxBitmapText extends FlxSprite
 	 * Set border's style (shadow, outline, etc), color, and size all in one go!
 	 * 
 	 * @param	Style outline style
-	 * @param	Color outline color in flash 0xRRGGBB format
+	 * @param	Color outline color in flash 0xAARRGGBB format
```

Comment:
Then again, border style (outline at least) already uses so many draw calls that an additional one wouldn't matter much.